go/src/koding/klient/machine/mount/anteroom.go

```diff
@@ -302,6 +328,22 @@ func (a *Anteroom) Unsync(path string) {
 	a.cursMu.Unlock()
 }
 
+// Pause prevets Anteroom from sending new events.
+func (a *Anteroom) Pause() {
+	atomic.StoreInt64(&a.paused, 1)
```

Comment:
Can we also a.idle.done(false)? If we wait for kd sync and pause concurrently, the original kd will gang indefinitely.